applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -726,19 +964,33 @@ namespace Kratos
         const bool CalculateResidualVectorFlag
         )
     {
-       KRATOS_ERROR << "You are calle to the CalculateAll from the base class for solid elements" << std::endl; 
+       KRATOS_ERROR << "You have called to the CalculateAll from the base class for solid elements" << std::endl; 
     }
     
     //***********************************************************************
     //***********************************************************************
 
     double BaseSolidElement::GetIntegrationWeight(
         const GeometryType::IntegrationPointsArrayType& ThisIntegrationMethod,
-        const unsigned int PointNumber,
+        const unsigned int point_number,
         const double detJ
         )
     {
-        return ThisIntegrationMethod[PointNumber].Weight() * detJ;
+        return ThisIntegrationMethod[point_number].Weight() * detJ;
+    }
+    
+    //************************************************************************************
+    //************************************************************************************
+    
+    void BaseSolidElement::UpdateKinematics(
```

Comment:
ok, i got the point. Two things:

   1. let's find out a name tha better expresses this
   2. @marandra the flow of the code is stl clear to you even after this change?